charts/gateway/values.yaml

```diff
@@ -20,25 +20,15 @@ image:
 # ```
 imagePullSecrets: []
 
-# -- Provide configuration for the Trino Gateway `dataStore` in `dataStoreSecret`. This node can
-# be left undefined if `dataStore` is defined under the config node. For production deployments
-# sensitive values should be stored in a Secret
-dataStoreSecret:
-  name: ""
-  key: ""
-
-# -- Provide configuration for the Trino Gateway `backendState` in `backendStateSecret`. This should
-# be used with health check configurations that require backend credentials. This node can
-# be left undefined if `dataStore` is defined under the config node.
-backendStateSecret:
-  name: ""
-  key: ""
-
-# -- Provide configuration for the Trino Gateway authentication configuration in `authenticationSecret`.
-# This node can be left undefined if `dataStore` is defined under the config node.
-authenticationSecret:
-  name: ""
-  key: ""
+# -- Provide a list of secrets and configmaps to mount into the init container as environment variables.
+# @raw
+# Example:
+# ```yaml
+# envFrom:
+#   - secretRef:
+#       name: password-secret
+# ```
+envFrom: []
```

Comment:
I like how this simplifies things. You added a test values.yaml file in the other PR: https://github.com/trinodb/charts/pull/261/files#diff-3314f8997bf0267424c20c4ed15ab01144cba76dd4bd383d7511c4cdcd1bc60f, but the `test.sh` script here already runs Gateway tests. Please add a `values.yaml` file for it here too, can be a basic one, just to verify that the chart renders and installs correctly with these sections filled up.

Reply:
I'll add a test once #261 is merged